qml/pages/PlayerPage.qml

```diff
@@ -22,7 +22,11 @@ Page {
 
     Connections {
         target: mediaPlayer
-        onPositionChanged: subtitlesText && subtitlesText.checkSubtitles()
```

Comment:
Was this causing some kind of silent error? I think there weren't any errors from this or were there?

Reply:
I just wanted to be sure that there are no additional errors. could be that this is not needed. Also prevents useless checks of subtitles if there are none.